Modules/Filtering/BiasCorrection/include/itkN4BiasFieldCorrectionImageFilter.h

```diff
@@ -405,14 +405,14 @@ class N4BiasFieldCorrectionImageFilter :
   // Parameters for deconvolution with Wiener filter
 
   unsigned int m_NumberOfHistogramBins{ 200 };
-  RealType     m_WienerFilterNoise{ 0.01 };
-  RealType     m_BiasFieldFullWidthAtHalfMaximum{ 0.15 };
+  RealType     m_WienerFilterNoise{ static_cast< RealType >( 0.01 ) };
```

Comment:
Why? There is still the implicit narrowing conversion.

Reply:
In the assignment scenario, the implicit conversion would not have been problematic? `static_cast`ing to the desired type kind of circumvents the point of type checking. If the assignment does generate an error or a warning, casting would be fine. But it might make sense to try the assignment.